### ISSUE TITLE
va-textarea Voice Over Bug ios

### ISSUE THREAD

**Issue by sixercjl** [department-of-veterans-affairs/vets-design-system-documentation#4762]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

When navigating to the new screen Voice Over (ios safari screen reader) will announce the text area account on screen loading. See video here:
https://coformaco.zoom.us/clips/share/RnjkOtlEQWerBLg9Ecaniw

and our internal ticket here:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/111889

## What I expected to happen

I am expecting the screen reader to only read when the component is focused and when the character count has been updated.

## Reproducing

- CSS Library version: "^0.25.0"
- component library version: "52.1.0"
- Device: iPhone 15
- Browser: Safari

**URL(s) where this bug occurs**

- Schedule new appointment: https://staging.va.gov/my-health/appointments/schedule/type-of-care
- The 'va-textarea' component lives on this page, but requires going through the flow starting at the link above: https://staging.va.gov/my-health/appointments/schedule/va-request/reason

**Steps to reproduce:**

1. Log into staging with test user on an iPhone using Safari.
2. Navigate to new health appointment, or go to https://staging.va.gov/my-health/appointments/schedule/type-of-care
3. (Any flow should work but I'll provide one example) Select 'Primary Care' and click 'Continue'.
4. If presented with choice of Community Care or VA Facility, choose VA and click 'Continue'.
5. Select any facility, click 'Continue'.
6. Select any appointment slots, no more than 3. I suggest activating Voice Over now if you haven't already. Click 'Continue'.
7. character count is announced

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other

**Comment by jeana-adhoc** [comment]
**Note:** I tried to triage his quickly to make sure it wasn't an implementation issue. And I think it might be more of a vo ios bug. This does not happen on Desktop Mac VO. JAWS and NVDA checks coming soon.

<img width="899" height="784" alt="screenshot of code" src="https://github.com/user-attachments/assets/3fbe84f1-0885-45ca-a970-fe1df53bd789" />

The h1 is not connected to the text area at all. There's no programmatic reason it should be announcing the character count aria-live region when the page loads. We'll have to investigate more.

Use the Judy Morrison user [from this list](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-vaos.md)
